tailwind.base.config.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
+  mode: 'jit',
   purge: {
-    enabled: process.env.TAILWIND_MODE === 'build',
+    enabled: true,
```

Comment:
Is that still needed with JIT mode?

Reply:
No we can get rid off it, it is the default I guess.
If we remove it, i guess we need to update all apps/libs tailwind config which extend this object, not a big deal.